docs/CONTRIBUTING.md

```diff
@@ -86,7 +86,21 @@ python onnx\backend\test\stat_coverage.py
 
 Some functionalities are tested with googletest. Those tests are listed in `test/cpp`, and include tests for shape inference, data propagation, parser, and others.
 
-To run them, first build ONNX with `-DONNX_BUILD_TESTS=1` or `ONNX_BUILD_TESTS=1 pip install -e .` and then run `.setuptools-cmake-build/onnx_gtests`.
+To run them, first build ONNX with `-DONNX_BUILD_TESTS=1` or `ONNX_BUILD_TESTS=1 pip install -e .`.
+
+The cpp tests require dynamically linking to built libraries.
+
+### Linux and MacOS
+```bash
+export LD_LIBRARY_PATH="./.setuptools-cmake-build/:$LD_LIBRARY_PATH
+.setuptools-cmake-build/onnx_gtests
+```
+
+### Windows
+```bat
+set PATH=./.setuptools-cmake-build/;%PATH%
+.setuptools-cmake-build/onnx_gtests
+```
```

Comment:
Not sure if this is right for windows. Don't have a local windows dev environment set up to test

Reply:
From my local Windows, I can run `.setuptools-cmake-build\Debug\onnx_gtests.exe` directly without setting additional PATH.